app/controllers/users_controller.rb

```diff
@@ -0,0 +1,12 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def destroy
+    @user = User.find(current_user)
```

Comment:
`find` expects an integer value for the ID whereas you're passing in a `User` object here.